[ROX-12430]: adding note about policy severity levels in RHACS

lahinson opened the pull request:
Version(s): 3.72.0

<!-- Version examples:
  * PR applies to all versions after a specific version (e.g. 4.8): 4.8+
  * PR applies to the in-development version (e.g. 4.11) and future versions: 4.11+
  * PR applies only to a specific single version (e.g. 4.10): 4.10
  * PR applies to multiple specific versions (e.g. 4.6-4.8): 4.6, 4.7, 4.8 --->

Issue: https://issues.redhat.com/browse/ROX-12430
<!--- Add a link to the Bugzilla, Jira, or GitHub issue, if applicable. --->

Link to docs preview: https://openshift-docs-git-rhacs-severity-levels-lahinson.vercel.app/openshift-acs/master/operating/default-security-policies.html
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. -->

<!-- NOTE:
Automatic preview functionality is currently not available.
  * OpenShift documentation team members (core and aligned) must include a link to a locally generated preview for PRs that update the rendered build in any way.
  * External contributors can request a generated preview from the OpenShift documentation team. --->

Additional information: This PR adds a note that explains that the policy severity levels in RHACS are different from the Red Hat Product Security severity levels. RHACS doesn't have QE, so no QE review was provided. This PR has been SME approved.

<!--- Optional: Include additional context or expand the description here.--->



<!--- Next steps after opening your PR:

* Ask for peer review from the OpenShift docs team:
  - For Red Hat associates: Ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
    * A link to the PR.
    * The size of the PR that the GitHub bot assigns (ex: XS, S, M, L, XL).
    * If there is urgency or a deadline for the review.
  - For community authors: Request a review by tagging @openshift/team-documentation in a GitHub comment.

  Slack is the quickest and preferred way to request a review.

* IMPORTANT:
  - All documentation changes must be verified by a QE team associate before merging.
  - Squash to one commit before submitting your PR for peer review.

* For more information about verifying your content, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#verification-of-your-content

* For more information about contributing to OpenShift documentation, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/contributing.adoc

Additional resources

The OpenShift docs repo adheres to the following style guides:

- OpenShift documentation guidelines (OSDOCS)
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc
- Red Hat Supplementary Style Guide (SSG)
  https://redhat-documentation.github.io/supplementary-style-guide/
- Modular Documentation Reference Guide (Mod Docs)
  https://redhat-documentation.github.io/modular-docs/
- IBM Style Guide (ISG)
  https://www.ibm.com/docs/en/ibm-style

You can log in to the ISG by using your @redhat.com id and single sign-on (SSO) credentials. --->

kcarmichael08 commented on operating/default-security-policies.adoc:
```diff
@@ -9,6 +9,15 @@ toc::[]
+
+The policy severity levels are Critical, High, Medium, and Low. Red Hat Product Security rates severity levels as Critical, Important, Moderate, and Low.
+
+While it is important to know both a policy's severity level understand the Red Hat Product Security severity levels, they are not related. For more information about the Red Hat Product Security severity levels, see link:https://access.redhat.com/security/updates/classification[Severity Ratings].
```
I think "to" is missing before "understand"

lahinson replied:
Good catch!

kcarmichael08 commented on operating/default-security-policies.adoc:
```diff
@@ -9,6 +9,15 @@ toc::[]
+====
+The severity levels for policies in {product-title} are different from the severity levels that are assigned by Red Hat Product Security.
+
+The policy severity levels are Critical, High, Medium, and Low. Red Hat Product Security rates severity levels as Critical, Important, Moderate, and Low.
```
May want to put "RHACS" before "policy severity levels" just to avoid confusion.

lahinson commented:
/label RHACS

bscott-rh commented on operating/default-security-policies.adoc:
```diff
@@ -9,6 +9,15 @@ toc::[]
 
+[NOTE]
+====
+The severity levels for policies in {product-title} are different from the severity levels that are assigned by Red Hat Product Security.
```
```suggestion
The severity levels for policies in {product-title} are different from the severity levels that Red Hat Product Security assigns.
```
Rewording suggestion to avoid passive voice.

lahinson commented:
/remove-label peer-review-done

lahinson commented:
/label peer-review-needed

lahinson commented:
@bscott-rh Thanks for the review! I removed the peer-review-done label, made the changes that you suggested, and added the peer-review-needed label back.

bscott-rh commented:
LGTM!

/remove-label peer-review-needed
/label peer-review-done

lahinson commented:
@bscott-rh I realized that I opened this PR against the wrong base, so I created PR #51660 against the correct base. I included the same peer-reviewed content from this PR in the new PR. Sorry to complicate things! Do you need to add the peer-review-done label to that one before I request merge review?

bscott-rh commented:
> @bscott-rh I realized that I opened this PR against the wrong base, so I created PR #51660 against the correct base. I included the same peer-reviewed content from this PR in the new PR. Sorry to complicate things! Do you need to add the peer-review-done label to that one before I request merge review?

no problem, I took a look at the new PR and added peer-review-done to it. You can go ahead with merge review
